integration/backward_compatibility.go

```diff
@@ -7,7 +7,7 @@ import "github.com/grafana/mimir/integration/e2emimir"
 // DefaultPreviousVersionImages is used by `tools/pre-pull-images` so it needs
 // to be in a non `_test.go` file.
 var DefaultPreviousVersionImages = map[string]e2emimir.FlagMapper{
-	"grafana/mimir:2.1.0": e2emimir.NoopFlagMapper,
+	"grafana/mimir:2.2.0": e2emimir.NoopFlagMapper,
```

Comment:
This is not correct. We want to test compatibility with old versions too. We should have 2.0.0, 2.1.0, and 2.2.0 here.